config/application.rb

```diff
@@ -10,7 +10,7 @@ module Smartflix
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.1
-
+    config.assets.paths << Rails.root.join("app", "assets", "fonts")
```

Comment:
If you do that, you can also drop the "fonts" from the load path here. But it's cool that you found out how to do this.